app/assets/javascripts/application.js

```diff
@@ -18,3 +18,22 @@
 //= require activestorage
 //= require turbolinks
 //= require_tree .
+
+$(document).ready(function() {
```

Comment:
I think you may want to use `turbolinks:load` - like this the code may not run unless you refresh the page (not website navigation)